### DISCUSSION TITLE
Does Geekdocs support inline code syntax in title?

### DISCUSSION THREAD

**Discussion by jamesbraza** [thegeeklab/hugo-geekdoc#646] [Q&A] [answered]
Does Geekdocs support inline code highlighting syntax in content titles?

```md
---
title: `foo` bar
---

Test
```

The above doesn't work.  I am wondering if there's an alternate way?

**Comment by xoxys** [comment, marked as answer]
That's not supported, sorry. The title is used in multiple places like menu sidebar and the browsers title bar, which doesn't support formatting at all. This theme is meant to be a simple theme, focussing on the content instead of a lot of features. Adding support for code fences in titles would require would add too much complexity.
